ci_framework/roles/operator_build/tasks/build.yml

```diff
@@ -14,17 +14,29 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: "{{ operator.name }} - Set default api path"
-  ansible.builtin.set_fact:
-    operator_api_path: "github.com/{{ cifmw_operator_build_org }}/{{ operator.name }}/api"
+- name: "{{ operator.name }} - Get base module name and set the api path"
+  block:
+    - name: "{{ operator.name }} - Get base module name from go.mod"
+      ansible.builtin.shell: |
+        grep {{ cifmw_operator_build_org }}/{{ operator.name }}/api go.mod | head -n 1 | xargs basename
+      args:
+        chdir: "{{ cifmw_operator_build_meta_src }}"
+      register: operator_base_module_name
+
+    - name: "{{ operator.name }} - Set default api path"
+      ansible.builtin.set_fact:
+        operator_api_path: "github.com/{{ cifmw_operator_build_org }}/{{ operator.name }}/{{ operator_base_module_name.stdout }}"
+  when:
+    - cifmw_operator_build_meta_build
```

Comment:
Dumb style thing, we are trying to put when clauses on top.

Reply:
thanks, done